.github/workflows/check-code.yml

```diff
@@ -0,0 +1,39 @@
+name: check-code
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+      - 'website/**'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - 'website/**'
+
+jobs:
+  build:
+    name: Check
+    runs-on: ubuntu-latest
```

Comment:
This is correct for our case?

Reply:
Yep, this is doing `make vet`, `make static` and other preventive checks on PR changes